source/meetings/general/2023-01-24.rst

```diff
@@ -0,0 +1,43 @@
+General Meeting 09:00 18-Sep-2023
+=================================
+
+Held at City Hall, Cardiff during PyCon UK 2023 and chaired by Owen Campbell.
+
+Trustees Present
+----------------
+
+Kristian Glass, Becky Smith, Owen Campbell, Vince Knight, Ann Barr
+
+
+Report
+------
+
+Owen Campbell presented the report to the members. Owen presented the
+achievements for the year:
+
+- PyCon UK ran again in 2022 after being postponed during the pandemic.
+  This is a notable achievement given the difficulties related to the pandemic.
+- The UKPA owns UKPA Trading Limited and for the first time the company was
+  ready for the conference. There is still some more progress to be made:
+  surplus funds from another entity need to be moved.
+- The UKPA supported the running of DjangoCon Europe. This is the first event
+  that was not PyCon UK that was supported by the UKPA.
+
+Trustees
+--------
+
+Becky Smith has to retire as a trustee according to clause
+
+.. TODO: Obtain clause number
```

Comment:
@meatballs I believe the missing clause is 13.1.4 and 13.1.5 which states:

> 
13.1.4
The members or the charity trustees may at any time decide to appoint a new charity trustee, whether in place of a charity trustee who has retired or been removed in accordance with clause 15, or as an additional charity trustee, provided that the limits specified in clause 12.4 on the number of charity trustees would not as a result be exceeded;

> 13.1.5
A person so appointed by the members of UKPA shall retire in accordance with the provisions of clauses 13.1.1 and 13.1.2. A person so appointed by the charity trustees shall retire at the conclusion of the annual general meeting next following the date of their appointment, and shall not be counted for the purpose of determining which of the charity trustees is to retire by rotation at that meeting.

Let me know if that's right and I'll add it in.

Reply:
Have added the clause in [58e9ffd](https://github.com/UKPythonAssociation/ukpa-internaldocs/pull/352/commits/58e9ffd37cb1d2af28277036ec654fa8faac89f3)